### ISSUE TITLE
Announced component doesn't work

### ISSUE THREAD

**Issue by voidsteed** [OfficeDev/office-ui-fabric-react#9055]
### Environment Information 
 
- __Package version(s)__: (6.170.0) 
- __Browser and OS versions__: (Chrome 75.0.3770.15 and windows 10 OS build 18362.86)

### Please provide a reproduction of the bug in a codepen:
 [ https://developer.microsoft.com/en-us/fabric#/controls/web/announced/lazyloading](url)

#### Actual behavior:
 Announced didn't say anything about `${total} of ${PHOTO_COUNT} photos loaded` in message when Narrator is on

 
#### Expected behavior:
 Announced say the message about `${total} of ${PHOTO_COUNT} photos loaded` when Narrator is on

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No)
 
Requested priority: (Blocking, High, Normal, Low) 
Blocking for accessibility
 
Products/sites affected: (if applicable)
 
![announcedBug](https://user-images.githubusercontent.com/2634606/57554146-7d056300-7325-11e9-888d-6f3413e0eb6c.gif)

**Comment by jdhuntington** [comment]
@voidsteed Thanks for the report. Are you able to share more about what platform, browser, and narrator you are using? I am unable to reproduce at the link you shared, and find instead that loading messages are announced. Thanks!

**Comment by KevinTCoughlin** [comment]
> @voidsteed Thanks for the report. Are you able to share more about what platform, browser, and narrator you are using? I am unable to reproduce at the link you shared, and find instead that loading messages are announced. Thanks!

Weird, I was talking to @voidsteed earlier and we both experienced this problem. But you're right, now it is working... Tested in Edge with the above link.

```shell
Pause loading, button,
status, 5 of 30 photos loaded
status, 11 of 30 photos loaded
```

**Comment by voidsteed** [comment]
@jdhuntington I'm using Windows 10 and Chrome.

**Comment by coryj627** [comment]
Hi Folks I've been helping @voidsteed. 

With NVDA 2019.1 and JAWS 201905 on Chrome 74 on Win 10

Both appear to be announcing the loading state at a staggered cadence of: 11, 17, 22, 28, 30 photos loaded.

JAWS seems to be announcing random numbers as well while it is focused on the button after activating it.

**Comment by voidsteed** [comment]
Confirmed Edge had it announced like this: 
`Scan
6 of 30 photos loaded
11 of 30 photos loaded
`
kinda random, maybe some logic in code. but I haven't checked it.

However it doesn't work in Chrome. I updated environment info in origin issue. And my browser and OS versions: (Chrome 75.0.3770.15 and windows 10 OS build 18362.86)

**Comment by jdhuntington** [comment]
Was able to repro this not functioning in Chrome. In my case, narrator announced "No item in view" roughly every 10 seconds.

**Comment by natalieethell** [comment]
@voidsteed thank you for reporting this issue! I'll take a look.

**Comment by natalieethell** [comment]
So unfortunately, Narrator + Chrome is not a combination that works very reliably. If you go to any web page on Chrome and start up Narrator, you'll likely hear the "No item in view" issue if you tab around a bit (you can even see this on the [Windows 10 complete guide to Narrator page](https://support.microsoft.com/en-us/help/22798/windows-10-complete-guide-to-narrator)).

This seems like more of a Narrator issue than a Fabric issue. I would recommend sticking to NVDA or JAWS to get read-out on Chrome and Windows 10 Narrator on Edge. @betrue-final-final correct me if I'm wrong here.

I can see how the Announced documentation doesn't make this super clear, though, and I think that's definitely an area that can be improved! We are mainly focused on Narrator + Edge scenarios, but I can update the documentation to make the recommended screen reader + browser combinations more apparent. 

@coryj627 regarding the staggered cadence, that's actually determined by the 10 second delay chosen for the example. Each photo loads every 2 seconds.

Let me know if that answers your question and if you have any other concerns!

**Comment by coryj627** [comment]
@natalieethell are you trying with the latest version of Chrome on Windows 10 May 2019 update (build 18362.30 or newer)? The May update and latest versions of Chrome introduced significant jumps in compatibility where Narrator plays much more nicely with browsers based on Chromium (in [preview since January](https://blogs.windows.com/windowsexperience/2019/01/03/announcing-windows-10-insider-preview-build-18309/)). Given that Edge is moving to a Chrome basis, it is detrimental to discount Narrator + Chromium.

**Comment by voidsteed** [comment]
Agree with @coryj627. If accessibility won't work in Chrome, how this component could work with future chromium version Edge?

**Comment by natalieethell** [comment]
@coryj627 @voidsteed definitely a valid point! I just reached out to someone on Edge who works closely with Narrator to ask about any existing compatibility issues. I'll update this issue as soon as I hear back from her.

**Comment by natalieethell** [comment]
Just got a response - the UIA (UI Automation) implementation in Chromium is not yet complete or turned on in new Edge by default. Once it is, though, these scenarios should work.

So, the new Edge is doing extra work to ensure that Narrator works properly with it. Narrator + Chrome, however, is not going to work as expected. Does that make sense? 

Let me know if I missed anything!

**Comment by coryj627** [comment]
This makes sense.

Have we tested with Google Chrome and Narrator on v1903? I know they did a ton of work there to build a proxy from MSAA to UIA.


—

Cory Joseph

@coryj206<https://twitter.com/coryj206> // in/CoryJoseph<https://linkedin.com/in/coryjoseph> // 503-395-0191<tel:+15033950191>
Three-word mission: Innovate with empathy…

________________________________

**Comment by natalieethell** [comment]
@coryj627 I haven't tested it on v1903 - I'm currently running v1809. If someone wants to test that out and report back here that would be great!

Since this doesn't seem to be a Fabric-specific bug, are we okay with closing out this issue?
